src/test/java/touro/chess/PawnPieceTest.java

```diff
@@ -0,0 +1,285 @@
+package touro.chess;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class PawnPieceTest {
+
+
+    @Test
+    public void getMovesBlack() {
+        // given
+        PawnPiece piece = new PawnPiece(new Location(5, 5), PieceColor.Black);
+
+        // when
+        List<Move> moves = piece.getMoves();
+
+        // then
+        assertEquals(3, moves.size());
+
+        //first move on list
+        Location locationExp1 = new Location(6, 5);
+        Location firstMove = moves.get(0).getTo();
+
+        assertEquals(locationExp1.toString(), firstMove.toString());
```

Comment:
The name "firstMove" implies that this is a `Move` object and not a `Location` object.